sdk/iot/Azure.Iot.Hub.Service/src/JobsClient.cs

```diff
@@ -3,23 +3,205 @@
 
 using System;
 using System.Collections.Generic;
-using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Iot.Hub.Service.Models;
 
 namespace Azure.Iot.Hub.Service
 {
     /// <summary>
-    /// Jobs Client place holder
+    /// Jobs client to support Import,Export and Scheduled Jobs.
     /// </summary>
     public class JobsClient
```

Comment:
I thought we had decided to split import/export jobs up from the scheduled jobs in terms of clients. Did we change our minds, or do I remember wrong?

One of the reasons is the get jobs and cancel job methods would be ambiguous.

Reply:
Yes we changed our minds to keep it in the same file. Does not stop us from changing our mind again.

Reply:
Okay, I'm curious how we plan to handle the ambiguity of those methods then.

Reply:
Honestly we've switched contexts so many times from this work that I don't even remember why we changed our minds :( I will revisit this design and schedule a meeting to discuss and come to a conclusion.

Reply:
From my analysis of the methods, looks like we have decided to do that using appropriate names - CancelImportExportJob and CancelScheduleJob. @prmathur-microsoft - Do you remember the discussion or decision?

Reply:
The notes I was keeping, explains our decision - [Notes](https://microsoft.sharepoint.com/teams/IotHub22/_layouts/OneNote.aspx?id=%2Fteams%2FIotHub22%2FSiteAssets%2FIot%20Hub%20Notebook&wd=target%28Hub%20Track2.one%7CC5D8B275-CA89-4E84-A686-F013BD35B146%2FIotHub%20meeting%20notes%7C5B2458B3-8C8A-494E-B99C-6DD00190FC44%2F%29)